src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -428,7 +428,7 @@ def run_container(container_name, image, flags=None, command=""):
 
 def remove_running_container(container_name):
     log.info(f"Removing Container {container_name}")
-    container_rm_cmd = f"podman-remote stop {container_name} && podman-remote rm {container_name}"
+    container_rm_cmd = f"podman-remote stop {container_name} && sleep 5 && podman-remote rm {container_name}"
```

Comment:
this should ignore the running condition of a container and just do the two actions in one go:
```suggestion
    container_rm_cmd = f"podman-remote rm --force {container_name}"
```

Reply:
In general ``sleep``s still doesn't guarantee stability, so it should be avoided

Reply:
ack , ok . how can we wait for stop action to be completed before move to remove ?  anyway i am chainging as you suggested.

Reply:
Reading the documentation I think it should do so, not sure why not:
```
$ podman-remote stop --help
Stop one or more containers

Description:
  Stops one or more running containers.  The container name or ID can be used.

  A timeout to forcibly stop the container can also be set but defaults to 10 seconds otherwise.
```

In principle there's the ``podman-remote wait`` command that can wait for a specific condition

Reply:
ack , so we know that 10 seconds its not enough base on the failure  and we can use force or use -t  flag for increasing timeout.
is it ok to add : f"podman-remote stop {container_name} && podman-remote rm --force {container_name}" ?

Reply:
It should work as well
Although redundant. Simply "podman rm --force" (without the stop command before) doesn't work?

Reply:
> It should work as well Although redundant. Simply "podman rm --force" (without the stop command before) doesn't work?

Yes it should work i just tried on my setup and looks like 10 second is not enough or we blocked on something.

[root@ocp-edge82 assisted-test-infra]# podman rm --force 46d30a105f25
WARN[0010] StopSignal SIGTERM failed to stop container load_balancer in 10 seconds, resorting to SIGKILL 
46d30a105f253a26c4052fe17fa7e99a017630f0717136c4f366301816ad82e3


I see that kill terminate the container immediatly ...
podman-remote kill 0823ceb41f0e

Not sure if we force or kill, we may have leftovers or maybe i am wrong.

Reply:
so replace ``podman-remote stop`` with ``podman-remote kill``. I think ``stop`` sends sigterm instead of sigkill which is a gracefulness we don't really need in this case